src/index.ts

```diff
@@ -24,6 +25,19 @@ const _isConnectedListeners = new Map<
   Types.NetInfoChangeHandler
 >();
 
+/**
+ * Sets configuration for NetInfo
+ */
+export function setNetInfoConfig(
+  newConfig: Partial<Types.NetInfoConfig>,
+): void {
+  Object.assign(netInfoConfig, newConfig);
```

Comment:
I presume you've tested that assigning to and imported `const` does actually change it for other files which have imported it?

It does seem a little "smelly" to me. I'd rather see the `internal/netInfoConfig` have a method to get and set the config and the `internal/internetReachability.ts` module fetches the config every time it needs it to make sure it's up-to-date.